state/bakerystorage/config.go

```diff
@@ -1,169 +0,0 @@
-	if err != nil {
-		return txn.Op{}, errors.Trace(err)
-	}
-	localUsersKeyBytes, err := json.Marshal(localUsersKey)
-	if err != nil {
-		return txn.Op{}, errors.Trace(err)
-	}
-
-	offersThirdPartyKey, err := bakery.GenerateKey()
-	if err != nil {
-		return txn.Op{}, errors.Trace(err)
-	}
-	offersThirdPartyKeyBytes, err := json.Marshal(offersThirdPartyKey)
-	if err != nil {
-		return txn.Op{}, errors.Trace(err)
-	}
-
-	return txn.Op{
-		C:      b.collection,
-		Id:     bakeryConfigKey,
-		Assert: txn.DocMissing,
-		Insert: &bakeryConfigDoc{
-			LocalUsersKey:              string(localUsersKeyBytes),
-			LocalUsersThirdPartyKey:    string(localUsersThirdPartyKeyBytes),
-			ExternalUsersThirdPartyKey: string(externalUsersThirdPartyKeyBytes),
-			OffersThirdPartyKey:        string(offersThirdPartyKeyBytes),
-		},
-	}, nil
-}
-
-// GetLocalUsersKey returns the key pair used with the local users bakery.
-func (b *bakeryConfig) GetLocalUsersKey() (*bakery.KeyPair, error) {
-	doc, err := b.bakeryConfigDoc()
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	return b.deserialiseKey(doc.LocalUsersKey, "local users key")
-}
-
-// GetLocalUsersThirdPartyKey returns the third party key pair used with the local users bakery.
-func (b *bakeryConfig) GetLocalUsersThirdPartyKey() (*bakery.KeyPair, error) {
-	doc, err := b.bakeryConfigDoc()
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	return b.deserialiseKey(doc.LocalUsersThirdPartyKey, "local users third party key")
-}
-
-// GetExternalUsersThirdPartyKey returns the third party key pair used with the external users bakery.
-func (b *bakeryConfig) GetExternalUsersThirdPartyKey() (*bakery.KeyPair, error) {
-	doc, err := b.bakeryConfigDoc()
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	return b.deserialiseKey(doc.ExternalUsersThirdPartyKey, "external users third party key")
-}
-
-// GetOffersThirdPartyKey returns the key pair used with the cross model offers bakery.
-func (b *bakeryConfig) GetOffersThirdPartyKey() (*bakery.KeyPair, error) {
-	doc, err := b.bakeryConfigDoc()
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	return b.deserialiseKey(doc.OffersThirdPartyKey, "offers third party key")
-}
-
-func (b *bakeryConfig) deserialiseKey(data, label string) (*bakery.KeyPair, error) {
-	if data == "" {
-		return nil, errors.NotValidf("empty " + label)
-	}
-
-	var keyPair bakery.KeyPair
-	err := json.Unmarshal([]byte(data), &keyPair)
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	logger.Debugf("using %s: %s", label, keyPair.Public.String())
-	return &keyPair, nil
-}
-
-func (b *bakeryConfig) bakeryConfigDoc() (*bakeryConfigDoc, error) {
-	var doc bakeryConfigDoc
-	controllers, closer := b.collectionGetter(b.collection)
-	defer closer()
-	err := controllers.FindId(bakeryConfigKey).One(&doc)
-	if err == mgo.ErrNotFound {
-		return nil, errors.NotFoundf("bakery config")
-	}
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	return &doc, nil
-}
```

Comment:
Can you delete the rest of the files in the bakerystorage directory yet?

Reply:
No, not yet. This is only the first step